packages/rune-games-cli/src/lib/validateGameFiles.ts

```diff
@@ -61,12 +61,8 @@ export async function validateGameFiles(
     errors.push({ message: `Game size must be less than ${maxSizeMb}MB` })
   }
 
-  const indexHtml = files.find(
-    (file) => file.path === "index.html" || file.path.endsWith("/index.html")
-  )
-  const logicJs = files.find(
-    (file) => file.path === "logic.js" || file.path.endsWith("/logic.js")
-  )
+  const indexHtml = findShortestPathFileThatEndsWith(files, "index.html")
```

Comment:
This does work, but can't we just do root/index.html, root/logic.js instead of searching/sorting?

Reply:
I think it's fine to enforce that the files are in root 👍

Reply:
Because validation is reused in cli, tango and also exists as a tango API endpoint, changing everything to now be aware of the root would require more work. maybe we can do this later, for now I just want to fix this pressing issue in a quickest way